packages/mc-dev-authentication/middlewares/create-login-middleware.js

```diff
@@ -0,0 +1,9 @@
+const createLoginMiddleware = (env) => (request, response, next) => {
+  if (Boolean(env.disableAuthRoutesOfDevServer) || env.servedByProxy) {
```

Comment:
Just for clarify. Either of them are booleans of strings?

Reply:
They're booleans usually. In theory the `disableAuthRoutesOfDevServer` can also be a string. Let me amend that. Thanks for pointing it out.